Views/Widget-Heading.liquid

```diff
@@ -21,6 +26,43 @@
     {% assign cssClasses = cssClasses | append: " " | append: visualLevel %}
 {% endif %}
 
-<{{ level }} id="{{ id }}" class="{{ cssClasses }} {{ Model.ContentItem | animation_css }}" style="{{ Model.ContentItem | animation_styles }}" {{ Model.ContentItem | animation_data_attributes }}>
+{% if isModal %}
+    {% assign cssClasses = cssClasses | append: ' js-iframe-modal' %}
+{% endif %}
+
+{% if linkUrl != blank %}

```

Comment:
Can we tidy up the tabbing within `{% if %}` statements and then change the formatting of how the attributes are added.

```
{% if linkUrl != blank %}
    <a href="{{ linkUrl | liquid }}"
    {% if openIn != null and isModal == false %} target="{{ openIn }}" {% endif %}
    {% if isModel %} data-iframe="true" {% endif %}
    {% if openIn == 'modal-wide' %} data-wide="true" {% endif %}
    {% if onClickEvent != blank %} onClick="{{ onClickEvent }}"  {% endif %}
    ...
{% else %}
    ...
{% endif %}
```